spec/rubocop/cop/lint/deprecated_open_ssl_constant_spec.rb

```diff
@@ -93,12 +93,18 @@
     RUBY
   end
 
-  it 'does not register an offense with cipher constant and send argument is a method' do
+  it 'does not register an offense with cipher constant and argument is a `send` node' do
```

Comment:
Can you tweak to the following?

```suggestion
  it 'does not register an offense with cipher constant and argument is a method call' do
```